lib/pymedphys/_mosaiq/helpers.py

```diff
@@ -271,8 +271,8 @@ def get_incomplete_qcls(cursor, location):
         """
         SELECT
             Ident.IDA,
-            Patient.Last_Name,
             Patient.First_Name,
+            Patient.Last_Name,
```

Comment:
There's still a few changes to this helpers script which could possibly break other tools (it's hard to know without Mosaiq automated testing :/). If these are needed could you undergo them in a separate helpers script within `_experimental`?